forms/user/registration_form.go

```diff
@@ -0,0 +1,71 @@
+package forms
+
+import (
+	"go-crawler-challenge/helpers"
+	"go-crawler-challenge/models"
+
+	"github.com/beego/beego/v2/core/validation"
+)
+
+type RegistrationForm struct {
+	Email           string `valid:"Required; MaxSize(100)"`
+	Password        string `valid:"Required; MinSize(6); MaxSize(12)"`
+	ConfirmPassword string `valid:"Required; MinSize(6); MaxSize(12)"`
+}
+
+var ValidationMessages = map[string]string{
```

Comment:
@Lahphim How about using constant here?
```suggestion
const ValidationMessages = map[string]string{
```

Reply:
can not use `const` with `map` 😭